README.md

```diff
@@ -191,6 +191,7 @@ You will need to have [Ruby](https://www.ruby-lang.org/en/) 2.3+ and
 [Bundler](https://bundler.io/) installed. Then:
 
 ```bash
+bundle config set --local path 'assets/bundle'
```

Comment:
@literally-bug-creator this doesn't look right. I'm not sure I can reproduce the issue: #68 It's something else, not what you fix here.